tests/backend/Functions.Tests/AnalyzeTests.cs

```diff
@@ -37,10 +41,14 @@ public class AnalyzeTests
             request.Content.Headers.Add("Content-Encoding", "gzip");
             request.Content.Headers.Add("Content-Type", "application/json");
 
-            var response = await Analyze.Run(request, new DoNothingTraceWriter());
+            var workflowQueue = Substitute.For<ICollector<WorkflowQueueMessage>>();
+            var response = await Analyze.Run(request, workflowQueue, NullLogger.Instance);
             var body = await response.Content.ReadAsStringAsync();
 
-            Assert.True(Guid.TryParse(body, out var _));
+            Guid submissionId;
+            Assert.True(Guid.TryParse(body, out submissionId));
+
+            workflowQueue.Received().Add(Arg.Is<WorkflowQueueMessage>(x => x.SubmissionId == submissionId.ToString() && x.Stage == WorkflowStage.Analyze));
```

Comment:
This should be a separate test. The function should return the guid in its response, and also queue a message.

Reply:
In this case the guid and queue are linked to the same function\action - so they're not that independent.  Also, in the actual test code, my verification simply adds one line to the test case to verify the queue message.  My suggestion is to leave this as-is for now.  If it proves to be an issue down the road, we can change if needed.  Thoughts?